src/shared-components/Card/ListItems/ListItemsSwitch.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+  Switch,
+} from '@mui/material';
+
+export const ListItemsSwitch = ({
+  title,
+  description,
+  isOn,
+  onToggle,
+}: {
+  title: string;
+  description: string;
+  isOn: boolean;
+  onToggle: () => void;
+}) => {
+  return (
+    <List>
```

Comment:
Remove list element, only keep listItem. Don't forget to clean imports.

Reply:
Fixed